src/lda.jl

```diff
@@ -91,7 +92,8 @@ function MulticlassLDAStats(cweights::Vector{T},
     MulticlassLDAStats(d, nc, cweights, tw, mean, cmeans, Sw, Sb)
 end
 
-function multiclass_lda_stats(nc::Int, X::DenseMatrix{T}, y::AbstractVector{Int}) where T<:Real
+function multiclass_lda_stats(nc::Int, X::DenseMatrix{T}, y::AbstractVector{Int};
+    covestimator=SimpleCovariance()) where T<:Real
```

Comment:
Incorrect indentation.

Reply:
Fixed. BTW, thank you for a quick review :+1: